### ISSUE TITLE
Login and Register button throws an error "Whoops something went wrong"

### ISSUE THREAD

**Issue by tomiwatech** [ammezie/support-ticket#7]
I noticed the login and register button throws and error  **Whoops something went wrong** . Am a newbie to laravel and i was just going through the demo before diving into the code. Thanks in Advance @ammezie

**Comment by ammezie** [comment]
Were you accessing on the Heroku link?

**Comment by luyandasiko** [comment]
@ammezie Yes the Heroku link, I was also trying the demo out and it gives that message. Thanks

**Comment by ammezie** [comment]
The Heroku app hasn't been maintained in a while. You might have to clone it locally to see how it works.

**Comment by marcelohama** [comment]
Not working for me as well. Same error, with two messages "Whoops something went wrong". I've tried with MAMP in localhost

**Comment by ammezie** [comment]
Set debug to `true` or check the `laravel.log` file to see the actual errors.
